scripts/validate_chainspecs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+ERR=0
+cargo build --release -p chainspec
+
+for spec in `ls ethcore/res/*.json`; do
```

Comment:
Unnecessary `ls`, just do `for spec in path/to/dir/*; do`. Iterating over `ls` output is unstable anyway. Also, backticks are deprecated and also smell and are bad.

Hey, instead of me telling you this stuff, go install [Shellcheck](https://github.com/koalaman/shellcheck/) since I think both of those notes are implemented as lints.